butte-examples/fbs/array.fbs

```diff
@@ -0,0 +1,24 @@
+/// Flatbuffers namespaces are defined as UpperCamelCase.
+namespace Array;
+
+/// A test entry with a value.
+table Entry {
+  value: uint64;
+}
+
+enum Value: ubyte {
+  A = 0,
+  B = 1
+}
+
+/// Different arrays.
+table ArrayTable {
+  table_entries: [Entry];
+  enum_entries: [Value];
+  int_entries: [int32];
+}
+
+/// Array of arrays.
+table ArrayOfArrayTable {
+  table_arrays: [[Entry]] (required);
```

Comment:
Sorry for the confusion, I did actually mean an array of `table`s that have at least one array field. In any case, happy to merge this as is.